modules/nf-core/deepcell/mesmer/meta.yml

```diff
@@ -0,0 +1,49 @@
+name: "deepcell_mesmer"
+description: Deepcell/mesmer segmentation for whole-cell
+keywords:
+  - imaging
+  - spatial_omics
+  - segmentation
+tools:
+  - "mesmer":
+      description: "Deep cell is a collection of tools to segment imaging data"
+      homepage: "https://github.com/vanvalenlab/deepcell-tf"
+      documentation: "https://github.com/vanvalenlab/intro-to-deepcell/tree/master/pretrained_models"
```

Comment:
"https://deepcell.readthedocs.io/en/master/"

There seems to be also this documentation available, @mashehu is it possibe to link both?